user/conditions-v0.md

```diff
@@ -4,7 +4,7 @@ layout: en
 ---
 
 > This page documents conditions v0, which is deprecated. Please opt in to the
-> new version v1 by adding `conditions: v1` to your `.travis.yml` file. The
+> new version v1 by adding conditions: v1 to your .travis.yml file. The
```

Comment:
Hmm, why did you remove the highlighting here?

Reply:
Accidentally. Fixed.